src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -275,16 +281,6 @@ void writeHistogramPDF(final SAMFileHeader readsHeader, final String inputFileNa
         final File histogramPlotPDF = new File(HISTOGRAM_PLOT_FILE);
         IOUtil.assertFileIsWritable(histogramPlotPDF);
 
-            /* Temporarily commented out because Picard R script for this purpose doesn't produce subtitle based on library
```

Comment:
Yes, Picard generates multiple plots if there are multiple groups. The comment is probably outdated and maybe I forgot to remove it? Anyway, they definitely should not be there. Thanks!